src/worker/keyman-api-service.ts

```diff
@@ -0,0 +1,36 @@
+import { keyboardDataObj, KeymanObj } from "./models";
+
+export class KeymanApi {
```

Comment:
Please capitalize "API":

```suggestion
export class KeymanAPI {
```

See: https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#general-keep-capitals-intact-in-names-with-acronyms-and-initialisms-in-camelcase